backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/PropertiesEventProcessor.java

```diff
@@ -1,187 +0,0 @@
-/*******************************************************************************
```

Comment:
Should'nt `org.eclipse.sirius.web.collaborative.forms.api.IPropertiesEventProcessor` be removed too?